core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/JupyterCodegenTests.kt

```diff
@@ -1,15 +1,40 @@
 package org.jetbrains.kotlinx.dataframe.jupyter
 
 import io.kotest.assertions.throwables.shouldNotThrowAny
+import io.kotest.matchers.should
 import io.kotest.matchers.shouldBe
 import io.kotest.matchers.types.shouldBeInstanceOf
 import org.intellij.lang.annotations.Language
+import org.jetbrains.kotlinx.dataframe.AnyFrame
+import org.jetbrains.kotlinx.dataframe.api.isNotEmpty
 import org.jetbrains.kotlinx.dataframe.columns.ValueColumn
 import org.jetbrains.kotlinx.jupyter.api.MimeTypedResult
 import org.jetbrains.kotlinx.jupyter.testkit.JupyterReplTestCase
 import org.junit.Test
 
 class JupyterCodegenTests : JupyterReplTestCase() {
+
+    @Test
+    fun `Don't inherit from data class`() {
+        @Language("kts")
+        val res1 = exec(
+            """
+            @DataSchema(isOpen = false)
+            data class A(val a: Int)
```

Comment:
hm, it also not clear why it 
```kotlin
**@DataSchema(isOpen = false)**
class A(val a: Int)
```
if isOpen is false, it shouldn't be here anyway

Reply:
maybe this is the real bug :)

Reply:
still need to check the private interface

Reply:
ah `private interface` breaks too cause `'public' member exposes its 'private-in-class' receiver type argument A`. It cannot compile the extension functions:
```kotlin
val ColumnsContainer<Line_3_jupyter.A>.a: DataColumn<Int> @JvmName("A_a") get() = this["a"] as DataColumn<Int>
val DataRow<Line_3_jupyter.A>.a: Int @JvmName("A_a") get() = this["a"] as Int
val ColumnsContainer<Line_3_jupyter.A?>.a: DataColumn<Int?> @JvmName("NullableA_a") get() = this["a"] as DataColumn<Int?>
val DataRow<Line_3_jupyter.A?>.a: Int? @JvmName("NullableA_a") get() = this["a"] as Int?
```
since `A` is private. But that's expected behavior I think.